src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -1644,6 +1647,103 @@ class SetClauseSegment(BaseSegment):
     )
 
 
+@ansi_dialect.segment()
+class CreateModelStatementSegment(BaseSegment):
+    """A BigQuery `CREATE MODEL` statement."""
+    type = 'create_model_statement'
+    # https://cloud.google.com/bigquery-ml/docs/reference/standard-sql/bigqueryml-syntax-create
+    match_grammar = Sequence(
+        'CREATE',
+        Sequence(
+            'OR',
+            'REPLACE',
+            optional=True
+        ),
+        'MODEL',
+        Sequence(
+            'IF',
+            'NOT',
+            'EXISTS',
+            optional=True
+        ),
+        Ref('ObjectReferenceSegment'),
+        Sequence(
+            'OPTIONS',
+            Bracketed(
+                Delimited(
+                    Sequence(
+                        Ref('ObjectReferenceSegment'),
```

Comment:
If you use `ParameterNameSegment` rather than `ObjectReferenceSegment` you shouldn't have a problem with the capitalisation warnings. 😁

I made the new segment for a very similar problem I was having in the snowflake dialect.

Reply:
Oh, nice! I'll make that change later tonight.